src/content/api/loyalty-programs.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Loyalty Programs
+description: Loyalty Program model and related endpoints
+nav:
+  path: API
+  order: 13
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+A loyalty program is a container for one or more user-facing components, to be surfaced in both Centrapay and third-party applications
+
+## Loyalty Program Model
+
+<Properties>
+  <Property name="id" type="string">
+    The unique identifier of the Loyalty Program.
+  </Property>
+
+  <Property name="name" type="string">
+    The display name of the Loyalty Program.
+  </Property>
+
+  <Property name="description" type="string">
+    The displayable description for the Loyalty Program.
+  </Property>
+
+  <Property name="mediaUploadId" type="string">
+    The id of the [Media Upload](/api/media-uploads/) image of the Loyalty Program.
+  </Property>
+
+   <Property name="img" type="string">
+    The img URL of the Loyalty Program.
+  </Property>
+
+  <Property name="accountId" type="string">
+    The [Account](/api/accounts/) that will own the Loyalty Program.
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    When the Loyalty Program was created.
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Loyalty Program.
+  </Property>
+
+  <Property name="test" type="boolean">
+    `true` if the Loyalty Program is for testing purposes only.
+  </Property>
+</Properties>
+
+---
+
+<Endpoint
+  path="/api/loyalty-programs"
+  filename="loyalty-programs-create"
+>
+  ## Create Loyalty Program <Badge type="experimental"/>
```

Comment:
You should remove this comment as it is not needed :)

Reply:
@Ray2126  It's a heading  :) - unsure why it appeared to be grey. We also have a heading for List Loyalty Programs by Account Id
![Screenshot 2024-02-09 at 5 17 36 AM](https://github.com/centrapay/centrapay-docs/assets/74471007/49ce0725-eaf4-46fd-a8c7-ef08fe84d854)